sebs/aws/triggers.py

```diff
@@ -123,3 +126,171 @@ def serialize(self) -> dict:
     @staticmethod
     def deserialize(obj: dict) -> Trigger:
         return HTTPTrigger(obj["url"], obj["api-id"])
+
+
+class QueueTrigger(Trigger):
+    def __init__(self, fname: str, deployment_client: Optional[AWS] = None):
+        super().__init__()
+        self.name = fname
+        self._deployment_client = deployment_client
+
+    @staticmethod
+    def typename() -> str:
+        return "AWS.QueueTrigger"
+
+    @property
+    def deployment_client(self) -> AWS:
+        assert self._deployment_client
+        return self._deployment_client
+
+    @deployment_client.setter
+    def deployment_client(self, deployment_client: AWS):
+        self._deployment_client = deployment_client
+
+    @staticmethod
+    def trigger_type() -> Trigger.TriggerType:
+        return Trigger.TriggerType.QUEUE
+
+    def sync_invoke(self, payload: dict) -> ExecutionResult:
+
+        self.logging.debug(f"Invoke function {self.name}")
+
+        # Init clients
+        lambda_client = self.deployment_client.get_lambda_client()
+        sqs_client = boto3.client('sqs', region_name=self.deployment_client.config.region)
+
+        serialized_payload = json.dumps(payload)
+
+        # Create queue
+        self.logging.debug(f"Creating queue {self.name}")
+
+        queue_url = sqs_client.create_queue(QueueName=self.name)["QueueUrl"]
```

Comment:
We should create the queue and add mapping once, when the trigger is created. Then, we store the URL and ARN once and use it during invocation.